src/pages/auth/alert/alert.js

```diff
@@ -0,0 +1,19 @@
+import "./alert.css";
+
+function ErrorAlert({ message, displayValue }) {
+  return (
+    <div
+      className="alert alert-bg-error dflex align-items-start"
+      style={{ display: displayValue ? "-webkit-box" : "none" }}
+    >
+      <span className="material-icons alert-info-icon alert-color-error mr-9">
+        cancel
+      </span>
+      <div>
+        <div className="alert-message">{`${message}.`}</div>
```

Comment:
{`${message}.`} can be simply written as {message}

Reply:
Done.